packages/generators/generators/lib/templates/js/plugin-package.json.hbs

```diff
@@ -8,7 +8,25 @@
     "kind": "plugin",
     "displayName": "{{titleCase pluginName }}"
   },
-  "dependencies": {},
+  "dependencies": {
+    "@strapi/design-system": "^1.6.3",
+    "@strapi/helper-plugin": "^4.6.0",
+    "@strapi/icons": "^1.6.3",
```

Comment:
Do we have to update these manually after each release? Same concern / question for the other dependencies.

Reply:
IMHO – no, the `^` will mean it will install the most recent version except for breaking changes – IF someone needs to lock a version they can do that themselves & we resolve the deps to our required version at the moment.

Reply:
In this `.hbs` file they will update too 🤔 ?

Reply:
> when it is installed in the plugin it will fetch the most recent version

Yes, sorry i should have been clearer.